src/styles/blocks/product-card.scss

```diff
@@ -0,0 +1,97 @@
+.product-card {
+  box-sizing: border-box;
+  width: 200px;
+  padding: 32px 16px 16px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  @include animation-hover(transform, scale(1.2));
+
+  &__images {
+    display: block;
+    width: 160px;
+    height: 134px;
+    margin: 0 auto 40px;
+  }
+
+  &__name {
+    font-weight: 500;
+    font-size: $font-size-m;
+    line-height: $line-h-l;
+    margin: 0 0 4px;
+    transition: color $time-for-animation;
+  }
+
+  &:hover &__name {
+    color: #34568b;
+  }
+
+  &__code {
+    font-weight: 400;
+    font-size: $font-size-s;
+    line-height: $line-h-s;
+    margin: 0 0 16px;
+    color: $second-color;
+  }
+
+  &__reviews {
+    margin-bottom: 24px;
+
+    display: flex;
+    align-items: flex-end;
+    justify-content: space-between;
+  }
+
+  &__reviews-paragraph {
+    font-weight: 400;
+    font-size: $font-size-s;
+    line-height: $line-h-s;
+    margin: 0;
+  }
+
+  &__price {
+    margin-bottom: 16px;
+
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+  }
+
+  &__price-label {
+    font-weight: 400;
+    font-size: $font-size-m;
+    line-height: $line-h-l;
+    color: $second-color;
```

Comment:
The variable `$second-color` is used without being defined in this file. Ensure that the variable is imported from a utilities or variables file if it's defined there, or define it in this file if it's missing.